Composer/packages/electron-server/src/main.ts

```diff
@@ -32,6 +32,12 @@ const getBaseUrl = () => {
   return `http://localhost:${serverPort}/`;
 };
 
+// set production flag
+if (app.isPackaged) {
```

Comment:
Is this the first place where app.isPackaged is set to true @tonyanziano . Can we move it outside directly inside main.ts not nested in any function?

Reply:
Or what if we use cross-env NODE_ENV=production inside the command yarn dist:full?

Reply:
I'm not sure I understand the wording of your question. Where would you like to move it to?

Reply:
Setting the variable at the script level doesn't work after testing. This is because `process.env` is scoped to the process at runtime, so attaching it to the `yarn dist:full` script would simply set it for the duration of building the production version, but at runtime it would be unset.